src/components/dueTimer.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import styled from 'styled-components'
+
+export default function DueTimer({ timeLeft }) {
```

Comment:
Maybe rename to `DueTime`, it doesn't really count time and just shows what is passed in.